File: app/src/main/java/com/example/sharewhatyoucanproject/ImageUploadInfo.java
package com.example.sharewhatyoucanproject;

public class ImageUploadInfo {

    public String imageName;

    public String imageURL;
    public String  imageDescription;
    //public float locationButton;

    public ImageUploadInfo() {

    }

    public ImageUploadInfo(String name, String url, String descr) {

        this.imageName = name;
        this.imageURL= url;
        this.imageDescription= descr;
        //this.locationButton = loc;

    }

    public String getImageName() {
        return imageName;
    }

    public String getImageURL() {
        return imageURL;
    }

    public String getImageDescription() {
        return imageDescription;
    }

    /*public float getLocationButton() {
        return locationButton;
    }*/
}

File: app/src/main/java/com/example/sharewhatyoucanproject/ImageUploadInfo.kt
package com.example.sharewhatyoucanproject

class ImageUploadInfo {
    var imageName: String? = null
    var imageURL: String? = null

    /*public float getLocationButton() {
         return locationButton;
     }*/
    var imageDescription: String? = null

    // public float locationButton;
    constructor() {}
    constructor(name: String?, url: String?, descr: String?) {
        imageName = name
        imageURL = url
        imageDescription = descr
        // this.locationButton = loc;
    }
}
